### ISSUE TITLE
Invalid Sound Card IRQ: 25

### ISSUE THREAD

**Issue by vanfanel** [crazii/SBEMU#104]
Hello again, @crazii 

I have this other small BMAX MiniPC, where trying to run SBEMU results in:

```
Sound card: Intel HDA
Invalid Sound card IRQ: 25, Trying to assign a valid IRQ...
Failed to assign a valid IRQ for sound card, abort.
```

Is it possible to support HDA on IRQ 25? (I understand that changing the IRQ is not possible.. is it?)

**Comment by crazii** [comment]
This problem is seen on VOGONS, the IRQ can be altered, but needs do something to the PCI IRQ router or APIC, that is planned to implemented later. But before that I can try to add more logs to see how it fails.

**Comment by vanfanel** [comment]
@crazii Yes, I talked about that on VOGONS too, and saw other persons had the same error.
I will gladly provide any logs you ask me for :)
